main.tf

```diff
@@ -100,6 +100,15 @@ resource "aws_security_group" "ecs" {
 resource "aws_ecs_cluster" "default" {
   name = var.name
   tags = var.tags
+
+  capacity_providers = aws_ecs_capacity_provider.default[*].name
+
+  dynamic "default_capacity_provider_strategy" {
+    for_each = aws_ecs_capacity_provider.default[*]
```

Comment:
Did you actually test this? Does this work?

Reply:
Yes, works both with and without the  `capacity_provider_asg_arn` variable

Reply:
Cool, yeah found the docs explaining this so all good indeed 👍🏻